packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,43 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  background-color: var(--gcds-button-text-only-default-background);
+
+  &
+  :not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
```

Comment:
Can you add a token for this, please? We are trying to avoid any hardcoded values in this repo.

Reply:
Just saw this too late, but yes definitely!